### ISSUE TITLE
Error message given for service gen failures in `ballerina openapi` is incomplete

### ISSUE THREAD

**Issue by pubudu91** [ballerina-platform/ballerina-lang#17737]
Consider a case where you try to generate a service from an OpenAPI file. The error message you get when the module name is not specified needs to be improved. Contains typos and should probably be phrased better.
```
$ ballerina openapi gen-service file-server.yaml
error: A module name is required to successfully generate the service from the provided OpenApi contract. 
E.g ballerina openapi gen-service <modulename>:<servicename>
```
The above error should be changed to something like below:
```
error: a module name and a service name is required to generate the service from the provided OpenAPI file. 
e.g., ballerina openapi gen-service <modulename>:<servicename>
```

**Comment by pubudu91** [comment]
It still gives the same error even params to the command are provided as asked.
```
$ ballerina openapi gen-service file-server.yaml file_server:fileService
error: A module name is required to successfully generate the service from the provided OpenApi contract. 
E.g ballerina openapi gen-service <modulename>:<servicename>
```

**Comment by pubudu91** [comment]
Apparently the correct form of the command is as follows:
```
$ ballerina openapi gen-service <module_name>:<service_name> <OpenAPI_file>
```
I feel that the above format of the command is not ideal. The main parameter/target of the gen-service command is the OpenAPI file. The module name and the service name are more like additional options given to the gen-service command. Isn't the previous format we had for this better? 
```
SYNOPSIS
    ballerina openapi [gen-service|gen-client] <openapifile>
        [-m <module-name>|--module <module-name>]
        [-o <path>|--output<path>]
```
We can add a `-s` option to the above.

**Comment by pubudu91** [comment]
Related issue: #17732

**Comment by NipunaMarcus** [comment]
looks like we haven't updated the help file for the latest revamping of the OpenAPI generator tooling... will update the help file ... 

About the `gen-service` command 

As we redesign the OpenAPI gen tools we followed the OpenAPI spec and decide to support this as the contract-first development. In this scenario, the user has the OpenAPI contract and what he wants to do is to start the development of the service documented in the contract. So that's where this tool comes in. As this is a fresh beginning and OpenAPI contract document only one service we decided to generate a fresh module and add the generated service and schema files into it. So the module name and service name is mandatory to keep it from conflicting with other existing services. 

Ballerina OpenAPI tooling Spec: https://docs.google.com/document/d/1x_z7I7sExrprSUHInhwLP36w6Dd0EgNq5LgFXukz6UA/edit?usp=sharing
